site/en/user_guide.md

```diff
@@ -227,25 +227,12 @@ ocrd-anybaseocr-crop  -I OCR-D-IMG -O OCR-D-BIN,OCR-D-IMG-BIN
 docker run --rm -u $(id -u) -v $PWD:/data -w /data -- ocrd/all:maximum ocrd-anybaseocr-crop  -I OCR-D-IMG -O OCR-D-BIN,OCR-D-IMG-BIN
 ```
 
-The [`parameter.json`](`parameter.json`) file can be created with the following command:
-
-```
-echo '{ "[parameter]": "[specification]" }' > [name of your param.json file]
-```
-
-Instead of calling a `parameter.json` file you can also directly
-write down the parameters when invoking a processor:
-
-```sh
--p '{"[parameter]": "[value]"}'
-```
-
-**Note:** If multiple parameters are necessary they have to be separated by a comma. (No comma after the last parameter!)
+**Note:** If multiple parameters are necessary, each of them has to be preceded by `-P`
```

Comment:
**Note:** If multiple parameter key-value pairs are necessary, each of them has to be preceded by `-P`